Solutions/Corvus.HighPerformance.Specs/ValueStringBuilderFeatures/Utf8/Utf8ValueStringBuilderStepDefinitions.cs

```diff
@@ -53,6 +53,34 @@ public void GivenIAttemptToReplaceWithAtIndexWithCount(
         this.Driver.Execute();
     }
 
+    [Given("I slice {int} characters from the start of the UTF-8 ValueStringBuilder")]
+    public void GivenISliceCharactersFromTheStartOfTheUtf8ValueStringBuilder(
+        int start)
+    {
+        this.Driver.AddOperation(new Utf8ValueStringBuilderTestDriver.AttemptSliceOperation(start, exceptionSteps));
+        try
+        {
+            this.Driver.Execute();
+        }
+        catch (ArgumentOutOfRangeException)
```

Comment:
Bearing in mind your `AttemptSliceOperation` catches and stores the exception, why do you need this?